build-info-extractor/src/main/java/org/jfrog/build/extractor/BuildInfoExtractorUtils.java

```diff
@@ -245,7 +293,7 @@ private static String getAdditionalPropertiesFile(Properties additionalProps, Lo
             filePath = additionalProps.getProperty(key);
             propFoundPath = "additionalProps.getProperty(" + key + ")";
         }
-        if (StringUtils.isBlank(filePath)) {
+        if (StringUtils.isBlank(filePath) && additionalProps != null) {
```

Comment:
Can it really be null? I looked at the 2 usages and I didn't found such scenario

Reply:
Removed